### PR TITLE
Add PR url to steelygh command

### PR DESCRIPTION
**Pull request by byxor**
People are using `.steelygh` a lot nowadays (given the sudden influx of shitty hacktoberfest plugins).

Sometimes I want a quick way to check the PRs, and the addition of this link will make that easier.

### CODE REVIEW

File: steely/plugins/github.py [hunk 1]
```diff
@@ -8,13 +8,23 @@
...
+API_URL = f'{PROTOCOL}://api.{DOMAIN}/repos/{REPOSITORY}'
+REGULAR_URL = f'{PROTOCOL}://{DOMAIN}/{REPOSITORY}'
+
+
 def handle_prs():
-    pulls = requests.get('https://api.github.com/repos/sentriz/steely/pulls').json()
-    return 'Senan, you have {} fucken pr\'s to merge'.format(len(pulls))
+    pulls = requests.get(f'{API_URL}/pulls').json()
+    message = f'Senan, you have {len(pulls)} fucken pr\'s to merge.'
+    message += '\n{REGULAR_URL}/pulls'
```
Comment (devoxel) [inline steely/plugins/github.py]: why not: 

```
f'Senan, you have {len(pulls)} fucken pr\'s to merge.\nREGULAR_URL/pulls'
```

Comment (byxor) [reply steely/plugins/github.py]: It goes over 80 characters. I don't mind breaking that rule though.

Comment (iandioch) [reply steely/plugins/github.py]: It's the same craic, Save The 80th

Comment (devoxel) [reply steely/plugins/github.py]: > resolved

**Comment by byxor** [comment]
@sentriz  I forgot to put an `f` in front of the string literal on line 22. Can you quickly fix?

**Pull request merged**